src/main/java/com/myworkspace/reportapp/repository/UserRepository.java

```diff
@@ -0,0 +1,21 @@
+package com.myworkspace.reportapp.repository;
+
+import com.myworkspace.reportapp.entity.customer.Employee;
+import com.myworkspace.reportapp.entity.customer.Manager;
+import com.myworkspace.reportapp.entity.customer.User;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface UserRepository extends JpaRepository<User, UUID> {
```

Comment:
Pagination for `findAllManagers`: If you plan to use pagination for the `findAllManagers` method in the future, you might want to update the method signature to include a `Pageable` parameter.

Reply:
corrected